microcosm_pubsub/handlers/publish_batch_message.py

```diff
@@ -22,7 +23,7 @@ def __init__(self, graph):
     def __call__(self, message):
         messages = message["messages"]
         for message in messages:
-            pubsub_message = PubsubMessage(
```

Comment:
I renamed `PubsubMessage` to `PublishedMessage` because it's an abstraction that's only used on the publisher side.